torch/csrc/jit/type.h

```diff
@@ -117,57 +116,123 @@ using TensorTypePtr = std::shared_ptr<TensorType>;
 // This node represents a single Tensor value with a specific size
```

Comment:
This is out of date now, irght?

Reply:
I think in general we need a more detailed description of the difference between TensorType/CompleteTensorType; which one to use in what situation; how to test if you have one or the other.